### PR TITLE
Worked on project with quinn 12/2/2023

### PR DESCRIPTION
**Pull request by EvergreenTor**

### CODE REVIEW

File: dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/Runtime.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                case "1":
+                    app.showAllData();
+                    break;
+
+                case "Q":
+                    running = false;
+                    break;
+
+                default:
+                    System.out.println("Your input is garbage, are you garbage too?");
```
Comment (learner-long-life) [inline dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/Runtime.java:24]: harsh ! we can work on cleaning inputs and recycling garbage.

File: dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/DataStorage.java [hunk 2]
```diff
@@ -21,6 +21,9 @@ public class DataStorage {
     private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
 
     // add public getters and setters for BST
+    public void addToSoundTree(double soundLevel, String date){
```
Comment (learner-long-life) [inline dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/DataStorage.java:24]: Very cool, it looks like you're using a binary search tree, and then recursively traversing all the left children to a leaf (for the minimum value) and all the right children to a leaf (for the maximum value).

**Comment by learner-long-life** [comment]
thanks for this work and effort @EvergreenTor @WestDav @aquinnallen

File: dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/DataStorage.java (hunk 2 above)
Comment (EvergreenTor) [reply dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/DataStorage.java:24]: that is correct

**Comment by WestDav** [comment]
This has been an honor

**Pull request merged**